zipkin-server/pom.xml

```diff
@@ -161,6 +161,11 @@
     </dependency>
 
     <dependency>
+      <groupId>org.springframework</groupId>
```

Comment:
not needed right?

Reply:
I was getting this exception while posting spans to `/api/v2/spans`
```
2017-10-03 11:08:28.605 ERROR 46979 --- [nio-9412-exec-2] o.a.c.c.C.[.[.[/].[dispatcherServlet]    : Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Handler dispatch failed; nested exception is java.lang.NoClassDefFoundError: org/springframework/util/concurrent/SettableListenableFuture] with root cause

java.lang.NoClassDefFoundError: org/springframework/util/concurrent/SettableListenableFuture
	at zipkin.server.ZipkinHttpCollector.validateAndStoreSpans(ZipkinHttpCollector.java:88) ~[classes!/:na]
	at zipkin.server.ZipkinHttpCollector.uploadSpansJson2(ZipkinHttpCollector.java:67) ~[classes!/:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_101]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_101]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_101]
```
This [stack overflow](https://stackoverflow.com/questions/41727213/nested-exception-is-java-lang-noclassdeffounderror-org-springframework-web-cont) suggested this solution. I couldn't get it working without adding this dependency.